src/platforms/ctxlink/WiFi_Server.c

```diff
@@ -558,10 +569,14 @@ static void app_wifi_callback(uint8_t msg_type, void *msg)
 	}
 
 	case M2M_WIFI_IP_ADDRESS_ASSIGNED_EVENT: {
-		t_wifiEventData *wifi_event_data = (t_wifiEventData *)msg;
-		if (wifi_event_data != NULL)
+		tstrM2MIPConfig *ip_config = (tstrM2MIPConfig *)msg;
```

Comment:
Beyond checking the pointer is non-NULL, is this local used for anything here? If not then the `if` should probably become `if (msg != NULL)`

Reply:
Like the above comment, the local is a useful debugging addition.